standard/clause_0_front_material.adoc

```diff
@@ -18,19 +18,19 @@ Given the intended use case of downloading data to a browser, there is no functi
 [abstract]
 == Abstract
 
-Based on JavaScript Object Notation (JSON), CoverageJSON is a format for publishing spatiotemporal data to the Web. The primary design goals are simplicity, machine and human readability and efficiency. While other use cases are possible, the primary CoverageJSON use case is enabling the development of interactive visual websites that display and manipulate environmental data within a web browser.
+Based on JavaScript Object Notation (JSON), CoverageJSON is a format for publishing geo-temporal data to the Web. The primary design goals are simplicity, machine and human readability and efficiency. While other use cases are possible, the primary CoverageJSON use case is enabling the development of interactive visual websites that display and manipulate environmental data within a web browser.
```

Comment:
I thought we change this from geo-temporal to spatiotemporal in an earlier iteration, because someone complained that "geo-temporal" wasn't a well-known term?

Reply:
@jonblower Agreed - @cnreediii was using an old copy. I missed that  and will correct.